### ISSUE TITLE
Create generic credentials for demo site in oclapi2

### ISSUE THREAD

**Issue by paynejd** [OpenConceptLab/ocl_issues#203]
Demo is reset daily, which means that to use the demo site, a user must always create a new user account. We would like to provide a generic user account for demo that is still reset daily but that is always available.

In the future, we can also provide some demo content and possibly multiple types of user accounts, but that is not a priority.

**Comment by paynejd** [comment]
@snyaggarwal Would this be easy to add for our oclapi2 deployment?

**Comment by snyaggarwal** [comment]
dependent on #373

**Comment by snyaggarwal** [comment]
@paynejd Do we need this?

**Comment by paynejd** [comment]
wont' fix
